include/kllvm/codegen/ProofEvent.h

```diff
@@ -60,6 +70,19 @@ class proof_event {
    */
   llvm::LoadInst *emit_get_proof_trace_writer(llvm::BasicBlock *insert_at_end);
 
+  /*
+   * Emit instructions to get the current value of the steps global variable,
+   * which counts the number of rewrite steps taken.
+   */
+  llvm::LoadInst *emit_get_steps(llvm::BasicBlock *insert_at_end);
+
+  /*
+   * Emit instructions to get the current value of the hproof_chunk_size global
```

Comment:
Is this `h` before `proof_chunk_size` a typo?

Reply:
yes, fixed in latest commit